entities/Search.ts

```diff
@@ -112,6 +112,14 @@ export class Search {
         return response as Promise<PixivAutoCompleteV2>
     }
 
+    /**
+     * Gets suggested candidates from web url that includes tag translation and access count.
+     */
+    public searchForCandidates = async (params?: PixivWebParams) => {
+        const response = await this.api.getWeb(`/rpc/cps.php?`, params)
```

Comment:
I don't think trailing `?` is needed axios already concatenates the params to the url.